regression/tests/helpers.py

```diff
@@ -90,3 +95,100 @@ def login(login_page):
         login_email = LoginHelper.get_login_email()
         login_password = LoginHelper.get_login_password()
         login_page.login(login_email, login_password)
+
+
+class LoginApi(object):
+    """
+    An Api to login the stage.
+    """
+    def __init__(self):
+        self.login_url = 'https://courses.stage.edx.org/login'
+        self.session = requests.Session()
+        self.session.auth = (
+            BASIC_AUTH_USERNAME,
+            BASIC_AUTH_PASSWORD
+        )
+        self.login_response = None
+
+    def check_response(self, response):
+        """
+        Check whether a response was successful. If not raise an exception
+
+        Arguments:
+            response: HTTP response object
+        """
+        if response.status_code != 200:
+            raise Exception(
+                'API request failed with following error code: ' +
+                str(response.status_code)
+            )
+
+    def post_headers(self, x_csrf):
+        """
+        Header which are to be used in the POST Requests
+
+        Arguments:
+            x_csrf: Cross site request forgery protection token.
+        """
+        return {
+            'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8',
+            'Accept': '*/*',
+            'X-Requested-With': 'XMLHttpRequest',
+            'Referer': self.login_url,
+            'X-CSRFToken': x_csrf,
+        }
+
+    def create_base_session(self):
+        """
+        Create a session with the host.
+        """
+        response = self.session.get(self.login_url)
+        self.check_response(response)
+        self.session.cookies = response.cookies
+        self.session.headers = self.post_headers(
+            response.cookies['csrftoken']
+        )
+
+    def login(self):
+        """
+        Login to the stage.
+        """
+        self.create_base_session()
+        login_post_url = 'https://courses.stage.edx.org/' \
```

Comment:
Same here - make this parameterized so that we can point to different environments.